### DISCUSSION TITLE
T-SQL physical name path on Linux

### DISCUSSION THREAD

**Discussion by wsmelton** [dataplat/dbatools#8711] [Contributors]
Should we be resolving this in the module since this is something T-SQL/SQL Server is doing and not us?

![image](https://user-images.githubusercontent.com/11204251/210281386-78b9d554-6960-44e7-a99f-3bd6b8586ad4.png)

The query can be modified to alter the path to a proper reference doing `REPLACE(mf.physical_name, '/\','/') AS PhysicalName`. I tested on SQL 2019 and this returned on Windows and Linux the proper path.

**Comment by potatoqualitee** [comment]
yes! i'd love for this to be fixed
